renderer/components/samples.tsx

```diff
@@ -62,9 +90,45 @@ const Samples = ({ sounds = [] }: { sounds: Sound[] }) => {
         await ipcService.getSounds(query, dispatch);
     };
 
+    const getCounts = async () => {
+        const getCount = async (key, opt) => {
+            if (!ipcService) return;
+            const query = filterToQuery({ ...filterState, [key]: opt });
+            const count = await ipcService.getSoundsCount(query);
+            return count;
+        };
+        const features = Object.keys(GROUPS);
+        const pairs = features.reduce((acc, feature) => {
+            const all = GROUPS[feature].map((value) => [feature, value]);
+            return acc.concat(all);
+        }, []);
+        const counts = {};
+        // Not a great implementation here since its essentially
+        // synchronous but the IPC event emitter returns the first response
+        // and returns the same result for all calls.
+        for (const [feature, value] of pairs) {
+            const count = await getCount(feature, value);
+            if (!counts[feature]) counts[feature] = {};
+            counts[feature][value] = count;
+        }
+        return counts;
+    };
```

Comment:
I think we would need to generate a unique response channel for each query in order to do it asynchronously.

Reply:
Another option could be to send a list of queries and stream the results back as is done for the analyzer.

Reply:
Yea it seems to perform fine with a large amount of sounds so I left it for now but we could proabbly do better here